src/navigator/navigator.js

```diff
@@ -0,0 +1,51 @@
+import ApioConsumer from '../consumer/apio-consumer';
+
+/**
+ * Class for navigating accross the API
+ */
+export default class Navigator {
+	/**
+	 * Creates a new Navigator
+	 * @param {ApioConsumer} consumer
+	 */
+	constructor(consumer) {
+		this.consumer = consumer;
+		this.navigationSteps = [];
+	}
+
+	getEntryPoint() {
+		const url = this.getEntryPointUrl();
+
+		const navigationStep = () => this.consumer.fetchResource(url);
+
+		this.navigationSteps.push(navigationStep);
+
+		return this;
+	}
+
+	getEntryPointUrl() {
+		const {baseUrl} = this.consumer.config;
+
+		return baseUrl + '/o/api';
```

Comment:
Why do you need the destructuring?

Reply:
Because... destructuring all the things!

Fixed in 98d1fb5